docs/tutorials/hobby_calibration.rst

```diff
@@ -124,32 +124,40 @@ Now, open Mission Planner and navigate to the Motor Test section under the Optio
 follow these steps:
 
     1. Power off the Vertiq module.
-    2. Set the duration in Motor Test to 120s. The precise number here isn't critically important, this is just meant to hold each command for a long enough time to make it easy to manually change them.
+    2. Set the duration in Motor Test to 25s. The precise number here isn't critically important, this is just meant to hold each command for a long enough time to make it easy to manually change them.
     3. Set the Throttle % to 100%.
     4. Click "Test All Motors" to start sending the 100% throttle commands.
     5. Power on the Vertiq Module, and wait for it to start playing the begin calibration song.
     6. Lower the Throttle % to 0% and click "Test All Motors." The module song should switch to the calibration-in-progress song.
-    7. Wait 5s, and then set the Throttle % to 50% and click "Test All Motors". The module should stop playing the calibration-in-progress song, indicating it completed calibration.
+    7. Wait 5s, and then set the Throttle % to 10% and click "Test All Motors". The module should stop playing the calibration-in-progress song, indicating it completed calibration.
     8. Lower the Throttle % to 0% and click "Test All Motors." Confirm that the module plays the arming song. Note that if you have changed the arming parameters of the module from their default settings it may not arm on a 0% throttle command, see the :ref:`manual_advanced_arming` section of the Feature Reference Manual for more information.
     9. Try out different Throttle % settings to confirm the module spins when armed.
     10. Power cycle the module and check the Calibrated Protocol, high-end, and low-end duration settings in the Control Center as described in `Checking and Trimming Calibration with Control Center`_ to confirm the calibration was successful.
 
-The figures below show what the Motor Test panel should look like at the beginning of the calibration procedure and what your calibration parameters in IQ Control Center should look like after a successful calibration,
-though the exact high and low durations may be different.
+You can see this whole procedure performed below.
 
-.. figure:: ../_static/tutorial_images/calibration_tutorial/mp_start_cal.JPG
-    :align: center
+.. raw:: html
+
+    <style type="text/css">
+    .center_vid {   margin-left: auto;
+                    margin-right: auto;
+                    display: block;
+                    width: 75%; 
+                }
+    </style>
+    <video class='center_vid' controls><source src="../_static/tutorial_images/calibration_tutorial/mission_planner_calibration.mp4" type="video/mp4"></video>
```

Comment:
Interesting, I didn't know you can add css styling here. Maybe in the future it would be better to move all the css styling to its own styles folder